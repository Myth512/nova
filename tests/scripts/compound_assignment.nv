// Case 1, expect: 15
a := 5
a += 10
print(a) 

// Case 2, expect: 6
b := 13
b -= 7
print(b)

// Case 3, expect: 18
c := 3
c *= 6
print(c)

// Case 4, expect: 2
d := 12
d /= 6
print(d)

// Case 5, expect: 1
e := 10
e %= 3
print(e)

// Case 6, expect: 16
f := 2
f ^= 4
print(f)
