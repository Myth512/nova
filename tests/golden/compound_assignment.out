15
6
18
2
1
16
